internal/host/validations_test.go

```diff
@@ -2605,4 +2605,134 @@ var _ = Describe("Validations test", func() {
 			Entry("is imported", &models.Cluster{ID: &clusterID, Imported: swag.Bool(true)}),
 		)
 	})
+	Context("Is MTU OK - CMN (Verify Reports Related to Machine Networks Only)", func() {
+		var (
+			cluster common.Cluster
+			host    models.Host
+		)
+		BeforeEach(func() {
+			cluster = hostutil.GenerateTestCluster(clusterID)
+			hostId, infraEnvId := strfmt.UUID(uuid.New().String()), strfmt.UUID(uuid.New().String())
+			mockProviderRegistry.EXPECT().IsHostSupported(commontesting.EqPlatformType(models.PlatformTypeVsphere), gomock.Any()).Return(false, nil).AnyTimes()
```

Comment:
Since the `isVSphereDiskUUIDEnabled` validation is executed in each unit test (each unit test runs all the validations), we specifically need the mock of `IsHostSupported` for the vSphere platform - https://github.com/openshift/assisted-service/blob/dffaec3a63661db97437147c2d0987dde482bfa5/internal/host/validator.go#L1674